ajax/libs/timedropper/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "timedropper",
+  "filename": "timedropper.min.js",
+  "version": "1.0",
+  "description": "timedropper jQuery time plugin",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/felicegattuso/timedropper.git"
+  },
+  "keywords": [
+    "timedropper",
+    "timepicker"
+  ],
+  "author": "Felice Gattuso",
+  "license": "MIT"
```

Comment:
The `LICENSE` section in http://felicegattuso.com/projects/timedropper/ shows that it's dual licenses. Could you confirm with author? @kennynaoh